### PR TITLE
Fix CMake CI

### PR DESCRIPTION
**Pull request by vrcomputing**
[Build](https://github.com/vrcomputing/qskinny/actions/runs/4605827886)

[CMake Build Matrix](https://github.com/vrcomputing/qskinny/actions/workflows/cmake.yml)

### CODE REVIEW

File: .github/workflows/cmake.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ name: CMake Build Matrix
 
 on:
   push:
-    branches: [cmake]
+    branches: [cmake_uwe]
```
Comment (vrcomputing) [inline .github/workflows/cmake.yml:11]: probably revert or change to master if final

File: .gitignore [hunk 2]
```diff
@@ -15,3 +15,4 @@ qskinny.pro.user
 html
 tags
 .vscode
+CMakeLists.txt.user
```
Comment (vrcomputing) [inline .gitignore:18]: this file is generated by Qt Creator

File: tools/svg2qvg/CMakeLists.txt [hunk 3]
```diff
@@ -33,9 +33,11 @@ if(BUILD_SVG2QVG_STANDALONE)
 
     target_compile_definitions(${target} PRIVATE QSK_STANDALONE)
     target_link_libraries(${target} PRIVATE Qt::Gui Qt::GuiPrivate)
+else()
+    target_link_libraries(${target} PRIVATE qskinny)
```
Comment (vrcomputing) [inline tools/svg2qvg/CMakeLists.txt:37]: this was required to remove "inconsistent dll linkage" error

**Comment by vrcomputing** [comment]
@uwerat you can probably squash these commits ( many irrelevant intermediate steps )

File: .github/workflows/cmake.yml [hunk 4]
```diff
@@ -371,35 +375,45 @@ jobs:
...
         if: startsWith(matrix.config.name, 'macOS')
+        env:
+          QT_DEBUG_PLUGINS: "1"
         run: |
           echo "starting iotdashboard"
-          DYLD_LIBRARY_PATH=./qskinny_install/lib:./qskinny_install/plugins/skins ./qskinny_build/examples/iotdashboard/iotdashboard.app/Contents/MacOS/iotdashboard -qwindowgeometry 1024x600+75+100 &
+          DYLD_LIBRARY_PATH=./qskinny_install/lib:./qskinny_install/plugins/skins ./qskinny_build/examples/bin/iotdashboard.app/Contents/MacOS/iotdashboard -qwindowgeometry 1024x600+75+100 --screenshot ${{ matrix.config.screenshot_filename }} &
           sleep 10
 
+      # TODO remove 'qskinny_build\skins\material3\Release' when skin install is fixed
```
Comment (vrcomputing) [inline .github/workflows/cmake.yml:385]: must be cleaned up when skins are being installed

**Comment by uwerat** [comment]
Merged into the cmake branch - CI tests succeeded

**Pull request closed without merging**